src/tarantool_schema.c

```diff
@@ -398,6 +398,21 @@ schema_add_space(
 	return -1;
 }
 
+int
+tarantool_schema_has_space_no(
```

Comment:
It seems it is not used.

Reply:
Confirmed that it is not used and removed.